src/main/java/com/iexec/sms/tee/config/SconeInternalServicesConfiguration.java

```diff
@@ -0,0 +1,33 @@
+package com.iexec.sms.tee.config;
+
+import com.iexec.common.tee.TeeEnclaveProvider;
+import com.iexec.sms.api.config.SconeServicesConfiguration;
+import com.iexec.sms.api.config.TeeAppConfiguration;
+import com.iexec.sms.api.config.TeeServicesConfiguration;
+import com.iexec.sms.tee.ConditionalOnTeeProvider;
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+import javax.validation.constraints.NotBlank;
+
+@Configuration
+@ConditionalOnTeeProvider(providers = TeeEnclaveProvider.SCONE)
+@Getter
+public class SconeInternalServicesConfiguration
+        extends SconeServicesConfiguration
+        implements TeeInternalServicesConfiguration{
+    public SconeInternalServicesConfiguration(
+            TeeAppConfiguration preComputeConfiguration,
+            TeeAppConfiguration postComputeConfiguration,
+            @Value("${tee.scone.las-image}")
+            @NotBlank(message = "las image must be provided")
+            String lasImage) {
+        super(preComputeConfiguration, postComputeConfiguration, lasImage);
+    }
+
+    @Override
+    public TeeServicesConfiguration getShareableConfiguration() {
```

Comment:
Why do we have `shareableConfiguration` here?
I feel it is just a custom conf (ex: gramine = shared + none, scone = shared +las)